bruno/collections/Rafiki/POS Service APIs/Initiate Payment.bru

```diff
@@ -0,0 +1,24 @@
+meta {
+  name: Initiate Payment
+  type: http
+  seq: 3
+}
+
+post {
+  url: http://localhost:3008/payment
+  body: json
+  auth: inherit
+}
+
+body:json {
+  {
+    "card": {
+      "walletAddress": "http://localhost:3000/accounts/gfranklin",
```

Comment:
I believe we should be passing in `https://cloud-nine-wallet/...` here, since the POS service will get it in the request and should find the wallet address using the shared rafiki network set up in the docker compose